src/classes/HierarchicalCollectionExample.cls

```diff
@@ -0,0 +1,33 @@
+public class HierarchicalCollectionExample {
+    public HierarchicalCollectionExample() {
+        Student__c student = new Student__c();
+        Map<String, Map<String, List<student>>> yogaClassMap = new Map<String, Map<String, List<student>>>();
+        
+        //1st Year List Of Students
+        List<student> Year1stListComputer = new List<student>();
+        Year1stListComputer = [SELECT id, Name, Roll_Number__c, Current_Year__c, Section_Stream__c FROM Student__c WHERE Current_Year__c=1 AND Section_Stream__c='Computer Science & Eng.'];
+        Map<String, List<student>> mapOfStudentwithStream1stYear = new Map<String, List<student>>();
+        mapOfStudentwithStream1stYear.put('Computer Science', Year1stListComputer);
+        
+        //1st Year List Of Students
+        List<student> Year1stListElectronic = new List<student>();
+        Year1stListElectronic = [SELECT id, Name, Roll_Number__c, Current_Year__c, Section_Stream__c FROM Student__c WHERE Current_Year__c=1 AND Section_Stream__c='Electrical Engineering'];
```

Comment:
When ever your line is being too long, give a line break;
eg:
 [SELECT id, Name, Roll_Number__c, Current_Year__c, Section_Stream__c FROM Student__c 
   WHERE Current_Year__c=1 AND Section_Stream__c='Electrical Engineering'];